### ISSUE TITLE
Adding logs to tf-gcp functions

### ISSUE THREAD

**Issue by tsuf239** [winglang/wing#4904]
### Feature Spec

When running tf-gcp tests, we can't see the logs we're creating using log.

### Use Cases

debugging

### Component

SDK

### Community Notes

* Please vote by adding a 👍 reaction to the issue to help us prioritize.
* If you are interested to work on this issue, please leave a comment.
* If this issue is labeled **needs-discussion**, it means the spec has not been finalized yet. Please reach out on the #dev channel in the [Wing Slack](https://t.winglang.io/slack).

**Comment by garysassano** [comment]
Could you provide more info about this?

**Comment by garysassano** [comment]
I think this is probably due to the fact that you are writing standard logs instead of [structured logs](https://cloud.google.com/functions/docs/monitoring/logging#writing_structured_logs).

There are probably two approaches to solve this I think:
- Manually write all logs as structured logs setting the `LogEntry` severity to `INFO`.
- [Reconfigure the standard log-writing interface to write to Cloud Logging](https://cloud.google.com/logging/docs/reference/libraries#write_standard_logs) (doesn't seem possible for Node.js client).

**Comment by garysassano** [comment]
I think this is probably due to the fact that you are writing standard logs instead of [structured logs](https://cloud.google.com/functions/docs/monitoring/logging#writing_structured_logs).

There are probably two approaches to solve this:
- Manually write all logs as structured logs setting the `LogEntry` severity to `INFO`.
- [Reconfigure the standard log-writing interface to write to Cloud Logging](https://cloud.google.com/logging/docs/reference/libraries#write_standard_logs) (doesn't seem possible for Node.js client).

**Comment by garysassano** [comment]
I think we already use Winston somewhere, so maybe an alternative could be using [@google-cloud/logging-winston](https://www.npmjs.com/package/@google-cloud/logging-winston)
 npm module.

**Comment by tsuf239** [comment]
Yes, the logs are created successfully in the GCP logs console, to display the logs on the user terminal we need to retrieve them somehow and print them- and that's what this issue is about :) 
please let me know if further information is needed

**Comment by Chriscbr** [comment]
Adding `contributing-experience` label since this will improve the feedback loop for anyone adding resources / capabilities to the GCP target
